openedx/FeedbackXBlock
# fix: large text for rating

### Description

This PR adds a small fix for when the rating text is too long for the options and increases the 'Number of ratings' to provide more space for feedback in the instructor tab

#### BEFORE
<img width="964" alt="Screen Shot 2024-03-18 at 6 43 09 PM" src="https://github.com/openedx/FeedbackXBlock/assets/134975835/e416e1bd-cb56-456b-94a1-985a338125b3">


#### NOW
<img width="974" alt="Screen Shot 2024-03-18 at 6 42 50 PM" src="https://github.com/openedx/FeedbackXBlock/assets/134975835/ba344c35-4ed1-4f56-820c-980e1659e50d">

### How to test it

1. Check this branch inside the XBlock
``

2. In your environment add the following setting [as doc](https://github.com/openedx/FeedbackXBlock?tab=readme-ov-file#tutor-configuration)
```yml
name: feedback-xblock-settings
version: 0.1.0
patches:
  openedx-common-settings: |
    FEATURES["ENABLE_FEEDBACK_INSTRUCTOR_VIEW"] = True
    OPEN_EDX_FILTERS_CONFIG = {
      "org.openedx.learning.instructor.dashboard.render.started.v1": {
        "fail_silently": False,
        "pipeline": [
          "feedback.extensions.filters.AddFeedbackTab",
        ]
      },
    }
```
3. From Studio, in your course > `Advanced Settings` > `Advanced Module List` > add "feedback" in the list.

4. In your course add a feedback component

5. Edit the component and put a long text for each `Likert option` save and check if the rating are in row
